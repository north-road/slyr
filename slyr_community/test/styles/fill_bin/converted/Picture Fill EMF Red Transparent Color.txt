[{'class': 'QgsSVGFillSymbolLayer',
  'pattern_width_unit': 'Point',
  'subsymbol': [{'class': 'QgsSimpleLineSymbolLayer', 'line_style': 'no'}],
  'svgFile': '<svg xmlns="http://www.w3.org/2000/svg" '
             'xmlns:svg="http://www.w3.org/2000/svg" '
             'xmlns:xlink="http://www.w3.org/1999/xlink" id="svg1" '
             'version="1.0">\n'
             ' <defs id="defs1">\n'
             '  <pattern height="6" id="EMFhbasepattern" '
             'patternUnits="userSpaceOnUse" width="6" x="0" y="0"></pattern>\n'
             ' </defs>\n'
             ' <image height="60" id="image1" preserveAspectRatio="none" '
             'transform="scale(1,-1)" width="76" x="0" y="-60" '
             'xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEwAAAA8CAIAAABD18u9AAAC0ElEQVRogc1aWXbCMAwsvNwBbsYVy83gFPQjPIfasjUajRPmr9TWYstac3q9Xj8Arvfn55+P2wVc3FsZIojLZtI5g4SqzZWI4L8G4HaBG1ElVz3d88bvJ3N1UZoBJQshRFtCJvoyXZzANwliFTR6BK16IIWycbxerGQGct9WsKRlcxCSxtzY7orSPF1+H5PcQCsQyKhnvZxVX+/PkZKgxSNsopK5fihE5GxSvN6fQl/HeeN8+CkqLO1PCEXich63S/TgTCslzqvreFROIg/idKr1/5KBEuhdWcGMdAYIFu84STv6AeOe6+dum4ui6zJNnCws5Yn7mFpP1er3s/krjU86xUVLfHVGQiNBTwrkpt3C+ATS2cyVcGI9INa7J1k+QSfChrAbEEK4nmwxtaiXQHCTBVzyHdrOoaskaI20BYKhNQoz4PtKuhJkEgm5quahB94k4vqJFLzNIjNvNdWSxOlyIk5qaq10fMeDPE6V1aliTEVH3P5ISjnpsATmOmYWMjxJR7fl+L5JbZgaXAieJxFO27SjrZFV0coUfj2WFYjMAXcKVTG0DJYmkc/4e53IKJEl09LkWM4gO2axhZBQVYE/mGRj5ZMIvZ3xrm2TluONI+l1NZ0BbTNaDuYmQ4daOTpwl/bUNiWLNGA7nOOHiF7WqFQVzCdd1xLN9eSuXqyk6nOPaDI4XgAp6XpwRAHifrgBUYtAqeVK5kZaba+kh1EyADIQJrSS7AepzgLmqhJuxpxvrCrqeEIj2qPQe/YB74o8jMN1NvUMtD/w13+sqsrOgKvzgarajSx595qjKZ8IoS1JgmiF/FtIaivu1km+ByAWjGF8M5CMY+2oNFrXy2e4hrkKi4Bo7h79+g28D+MmtXN1Ye1Lk7Lf5KQxEz0LmlhPapNpScOao6OZan05/BAS/ZLwC9t24klzdOU+mJgM7Knn+FiZelI1dVO9dpfdHxqvcct3dZOrAAAAAElFTkSuQmCC"></image>\n'
             '</svg>',
  'width': '76'},
 {'capstyle': 'round',
  'class': 'QgsSimpleLineSymbolLayer',
  'joinstyle': 'round',
  'line_color': '110,110,110,255,rgb:0.4313725,0.4313725,0.4313725,1',
  'line_width': '1',
  'line_width_unit': 'Point'}]
