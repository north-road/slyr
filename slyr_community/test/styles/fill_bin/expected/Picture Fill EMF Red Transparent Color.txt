{'levels': [{'angle': 0.0,
             'color_background': {'B': 0,
                                  'G': 0,
                                  'R': 0,
                                  'dither': False,
                                  'is_null': True,
                                  'stream_offset': '0x3777',
                                  'type': 'RgbColor',
                                  'version': 1},
             'color_background_model': 'rgb',
             'color_foreground': None,
             'color_foreground_model': None,
             'color_transparent': {'B': 0,
                                   'G': 0,
                                   'R': 255,
                                   'dither': True,
                                   'is_null': False,
                                   'stream_offset': '0x37b6',
                                   'type': 'RgbColor',
                                   'version': 1},
             'color_transparent_model': 'rgb',
             'enabled': True,
             'locked': False,
             'offset_x': 0.0,
             'offset_y': 0.0,
             'outline': {'color': {'B': 110,
                                   'G': 110,
                                   'R': 110,
                                   'dither': True,
                                   'is_null': False,
                                   'stream_offset': '0x37f7',
                                   'type': 'RgbColor',
                                   'version': 1},
                         'enabled': True,
                         'line_type': 'solid',
                         'locked': False,
                         'stream_offset': '0x37e5',
                         'symbol_level': 0,
                         'type': 'SimpleLineSymbol',
                         'version': 1,
                         'width': 1.0},
             'picture': {'content': b'AQAAAHQAAABzAQAA0f3//78BAAAN/v//Si0AAMO7'
                                    b'//+RNgAAFsP//yBFTUYAAAEADDcAAAsAAAACAAAA'
                                    b'DQAAAFgAAAAAAAAAAAQAAAADAABAAQAA8AAAAEEA'
                                    b'ZABvAGIAZQAgAFMAeQBzAHQAZQBtAHMAAAARAAAA'
                                    b'DAAAAAgAAAASAAAADAAAAAEAAAAKAAAAEAAAAAAA'
                                    b'AAAAAAAADAAAABAAAAAAAAAAAAAAAAkAAAAQAAAA'
                                    b'rAIAABwCAAALAAAAEAAAAOQAAAC0AAAAJAAAACQA'
                                    b'AAAAAEBAAAAAgAAAAAAAAEDAAACLRACA0cQCAAAA'
                                    b'UQAAAOg1AABzAQAA0f3//78BAAAN/v//AAAAAMT/'
                                    b'//8AAAAAAAAAAEwAAAA8AAAAUAAAACgAAAB4AAAA'
                                    b'cDUAAAAAAAAgAMwATAAAADwAAAAoAAAATAAAADwA'
                                    b'AAABABgAAAAAAHA1AAATCwAAEwsAAAAAAAAAAAAA'
                                    b'////////////////////////////////////////'
                                    b'////////////6KgX6KgX6KgX6KgX////////////'
                                    b'////////////////////////////////6KgX6KgX'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////6KgX6KgX6KgX////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////6KgX6KgX6KgX6KgX////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////6KgX6KgX'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////////////6KgX6KgX6KgX6KgX////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////6KgX6KgX////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX////////'
                                    b'////////////////////////////////6KgX6KgX'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////////////////6KgX6KgX////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////6KgX6KgX6KgX////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX6KgX////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////6KgX6KgX////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX6KgX////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX6KgX////////////////////////////'
                                    b'6KgX6KgX6KgX////////////////////////////'
                                    b'////////6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX6KgX6KgX////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX////////'
                                    b'////////////////////////6KgX6KgX6KgX////'
                                    b'////////////////////////6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX6KgX6KgX////////////////////////'
                                    b'////////6KgX6KgX6KgX////////////////////'
                                    b'////////6KgX6KgX6KgX////////////////////'
                                    b'////6KgX6KgX6KgX////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'6KgX6KgX6KgX////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX6KgX////'
                                    b'////////////////////////////////6KgX6KgX'
                                    b'6KgX6KgX6KgX////////////6KgX6KgX6KgX6KgX'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX////////////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX6KgX////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////6KgX6KgX6KgX6KgX6KgX6KgX6KgX////'
                                    b'////////////////////////////////////////'
                                    b'////6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////6KgX6KgX6KgX////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX////////////////////////////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX6KgX6KgX////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////6KgX6KgX6KgX6KgX'
                                    b'6KgX////////////////////////////////////'
                                    b'////////////////////////////6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX6KgX////////////////'
                                    b'////////////////////////////////6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'////////////////////////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX6KgX////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX////////////////////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX6KgX6KgX////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'////////////////////////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX6KgX////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'6KgX6KgX6KgX////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX////////////////////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX6KgX6KgX////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////6KgX6KgX6KgX6KgX////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////6KgX6KgX6KgX6KgX6KgX6KgX6KgX////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////6KgX6KgX6KgX6KgX6KgX'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'6KgX6KgX6KgX6KgX6KgX////////////////////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////6KgX6KgX////////////////////////'
                                    b'////////////6KgX6KgX6KgX6KgX6KgX////////'
                                    b'////////////////////6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX////////////////////////////////////'
                                    b'////////6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////6KgX6KgX'
                                    b'6KgX6KgX6KgX////////////////////////////'
                                    b'////6KgX6KgX6KgX6KgX6KgX////////////////'
                                    b'////////////////////////6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX////////'
                                    b'////////////////////6KgX6KgX6KgX6KgX6KgX'
                                    b'////////////////////////////////////////'
                                    b'////6KgX6KgX6KgX6KgX6KgX6KgX////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////6KgX6KgX////////////////////////'
                                    b'////////////////////////////6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX////////////////////////////'
                                    b'////6KgX6KgX6KgX6KgX6KgX////////////////'
                                    b'////////////////////////////6KgX6KgX6KgX'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////6KgX6KgX6KgX'
                                    b'6KgX////////////////////////////////////'
                                    b'////////6KgX6KgX6KgX6KgX6KgX6KgX6KgX////'
                                    b'////////////////////////////6KgX6KgX6KgX'
                                    b'6KgX////////////////////////////////////'
                                    b'////////////////6KgX6KgX////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////6KgX6KgX6KgX6KgX////////////'
                                    b'////////////////////////////6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX////////////////////'
                                    b'////////6KgX6KgX6KgX6KgX////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////////////6KgX6KgX6KgX6KgX6KgX6KgX////'
                                    b'////////////////////////////////6KgX6KgX'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////6KgX6KgX6KgX////////'
                                    b'////////////////////////////////////////'
                                    b'6KgX6KgX6KgX6KgX////////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////6KgX6KgX////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'6KgX6KgX6KgX6KgX6KgX6KgX////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////6KgX6KgX6KgX////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX////////'
                                    b'////////////////6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX////////////////////////'
                                    b'////////////////////////////////6KgX6KgX'
                                    b'6KgX////////////////////////////////////'
                                    b'6KgX6KgX6KgX6KgX6KgX////////////////////'
                                    b'6KgX6KgX6KgX6KgX////////////6KgX6KgX////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'////////////////////////////////////////'
                                    b'////////6KgX6KgX6KgX6KgX6KgX////////////'
                                    b'////////////////////////6KgX6KgX6KgX6KgX'
                                    b'6KgX////////////////6KgX6KgX6KgX6KgX////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX////////6KgX6KgX6KgX////////////////'
                                    b'////////////////////////////6KgX6KgX6KgX'
                                    b'6KgX////////////////////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX6KgX////'
                                    b'////6KgX6KgX6KgX////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////6KgX6KgX6KgX6KgX6KgX////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX6KgX////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX6KgX6KgX////////////////////////'
                                    b'////////////////////6KgX6KgX6KgX////////'
                                    b'////////////////////////////6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////6KgX6KgX6KgX6KgX6KgX////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////6KgX6KgX6KgX6KgX6KgX'
                                    b'////////////////////////////////6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX6KgX////////////////'
                                    b'////////////6KgX6KgX6KgX6KgX6KgX////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////6KgX6KgX6KgX6KgX'
                                    b'6KgX////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'6KgX6KgX6KgX6KgX6KgX////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX////////////////////////6KgX6KgX'
                                    b'6KgX6KgX6KgX////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////6KgX6KgX6KgX6KgX6KgX////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////6KgX6KgX6KgX6KgX'
                                    b'////////////////////////////////////////'
                                    b'6KgX6KgX6KgX6KgX6KgX6KgX////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////6KgX6KgX6KgX6KgX6KgX////////////////'
                                    b'////////////////////////6KgX6KgX6KgX////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX6KgX6KgX////////////////////////'
                                    b'////////////////////////////////6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////6KgX6KgX6KgX'
                                    b'6KgX////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////6KgX6KgX6KgX6KgX6KgX6KgX////'
                                    b'////////////////////////////////////////'
                                    b'////6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX6KgX////////////////////////////'
                                    b'////////////////////////6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////6KgX6KgX6KgX////////'
                                    b'////////////////////////////////////////'
                                    b'////////6KgX6KgX6KgX6KgX6KgX////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////6KgX6KgX6KgX////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////6KgX6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////6KgX6KgX6KgX6KgX////////////////'
                                    b'////////////////////////////////////////'
                                    b'////6KgX6KgX6KgX6KgX////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////////////////////////////6KgX6KgX6KgX'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////6KgX6KgX6KgX6KgX6KgX////6KgX6KgX6KgX'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////6KgX6KgX6KgX6KgX6KgX6KgX////////////'
                                    b'////////////////////////////////////////'
                                    b'////////6KgX6KgX6KgX6KgX////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////6KgX6KgX6KgX6KgX'
                                    b'////////////////////6KgX6KgX6KgX////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX////////////////////////////'
                                    b'////////////////////////////////6KgX6KgX'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX6KgX////////////////'
                                    b'////////6KgX6KgX////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////6KgX6KgX6KgX6KgX6KgX////////////'
                                    b'////////////////////////////6KgX6KgX6KgX'
                                    b'////////6KgX6KgX6KgX6KgX////////////////'
                                    b'////////////////////////////////////////'
                                    b'////6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX////'
                                    b'////////////////////////////////6KgX6KgX'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////6KgX6KgX6KgX'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////////////6KgX6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX6KgX////////////////'
                                    b'////////////////////////6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX////////////////////////'
                                    b'////////////////6KgX6KgX////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////6KgX6KgX6KgX6KgX////////////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX////////////////////'
                                    b'////////6KgX6KgX6KgX6KgX6KgX////////////'
                                    b'////////////////////////////////////////'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////////////////////////////////6KgX6KgX'
                                    b'6KgX6KgX6KgX////////////////////////////'
                                    b'////////////////////////////6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX////////////////6KgX6KgX6KgX'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////6KgX6KgX6KgX////////'
                                    b'////////////////////////////////////////'
                                    b'////////6KgX6KgX6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'////////////////////////////////////////'
                                    b'////////////6KgX6KgX6KgX6KgX6KgX////////'
                                    b'////////////////////6KgX6KgX6KgX////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX6KgX6KgX////////////////////////'
                                    b'////////////////////////////6KgX6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX6KgX6KgX////////////'
                                    b'////////////////////////////////////////'
                                    b'6KgX6KgX6KgX6KgX////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX6KgX////'
                                    b'////////////////////////////////////////'
                                    b'////6KgX6KgX6KgX6KgX6KgX6KgX////////6KgX'
                                    b'6KgX6KgX6KgX6KgX6KgX////////////////////'
                                    b'////////////////////////////6KgX////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'6KgX6KgX6KgX6KgX////////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'6KgX6KgX////////////////////////6KgX6KgX'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'6KgX6KgX6KgX6KgX6KgX////////////////////'
                                    b'////////////////////6KgX6KgX6KgX6KgX////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////6KgX6KgX////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX6KgX6KgX'
                                    b'////////////////////////////////////////'
                                    b'////////////6KgX6KgX6KgX////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////6KgX6KgX6KgX6KgX6KgX////////////'
                                    b'////////////////////////////////6KgX6KgX'
                                    b'6KgX6KgX6KgX////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'6KgX6KgX6KgX6KgX////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////6KgX6KgX'
                                    b'6KgX////////////////////////////6KgX6KgX'
                                    b'6KgX6KgX6KgX////////////////////////////'
                                    b'////////////6KgX6KgX6KgX6KgX6KgX////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////6KgX6KgX6KgX6KgX'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////6KgX6KgX6KgX6KgX6KgX////////////'
                                    b'////////////////////6KgX6KgX6KgX6KgX6KgX'
                                    b'////////////////////////////////6KgX6KgX'
                                    b'6KgX6KgX6KgX6KgX////////////////////////'
                                    b'////////////////////////////////////////'
                                    b'////////6KgX6KgX6KgX6KgX6KgX////////////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////6KgX6KgX6KgX'
                                    b'6KgX6KgX////////////////////////////////'
                                    b'////////6KgX6KgX6KgX6KgX////////////////'
                                    b'////////////////6KgX6KgX6KgX6KgX6KgX////'
                                    b'////////////////////////////////////////'
                                    b'////////////////////////////////////6KgX'
                                    b'6KgX6KgX6KgX////////////////////////////'
                                    b'IwAAACAAAAAAAIA/AAAAAAAAAAAAAIA/AAAAAAAA'
                                    b'AAAOAAAAFAAAAAAAAAAQAAAAFAAAAA==',
                         'type': 'EmfPicture',
                         'version': None},
             'scale_x': 1.0,
             'scale_y': 1.0,
             'separation_x': 0.0,
             'separation_y': 0.0,
             'stream_offset': '0x4d',
             'swap_fg_bg': False,
             'symbol_level': 0,
             'type': 'PictureFillSymbol',
             'version': 8}],
 'stream_offset': '0x0',
 'symbol_level': 0,
 'type': 'MultiLayerFillSymbol',
 'version': 2}
